tests/blueprint_specific_tests/search_bp_tests/test_search_endpoints.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for endpoints in the search blueprint
+"""
+import pytest
+from urllib.parse import urlparse
+from flask import url_for
+from flask_login import current_user, login_user, logout_user
+
+
+def test_pipelines_route(test_client):
+    """
+    GIVEN calling the route "/search"
```

Comment:
it's a little confusing that `/search` is the endpoint but the test is named "test_pipelines_route" here..

Reply:
Ah, ok. That's a typo which I missed.